ark/utils/data_utils.py

```diff
@@ -85,13 +89,21 @@ def label_cells_by_cluster(fovs, all_data, label_maps, fov_col=settings.FOV_ID,
         relabeled_img_array = relabel_segmentation(labeled_img_array, labels_dict)
         img_data.append(relabeled_img_array)
 
-    return xr.DataArray(img_data, coords=[fovs, range(img_data[0].shape[0]),
-                                          range(img_data[0].shape[1])],
-                        dims=["fovs", "rows", "cols"])
+    return xr.DataArray(
+        img_data,
+        coords=[fovs, range(img_data[0].shape[0]), range(img_data[0].shape[1])],
```

Comment:
I think changes like this, making sure data structures (list, dict, tuple, etc) that previously spanned multiple lines are now on the same line increases readability